### ISSUE TITLE
Wrong title in folder_contents (Plone Classic)

### ISSUE THREAD

**Issue by verena-rh** [plone/Products.CMFPlone#4011]
## BUG/PROBLEM REPORT (OR OTHER COMMON ISSUE)

In Plone Classic folder_contents I found the following problem:
If you change to another folder via the breadcrumbs above the table in folder_contents, the title of the previous folder is still displayed as content title.

### What I did:

* Create two folders folder1 and folder2 in the Plone root
* Go to folder_contents in folder1
* Use the breadcrumbs of folder_contents to navigate to root and then to folder2

### What I expect to happen:

The title of folder2 is displayed as content title.

### What actually happened:

The title of folder1 is still displayed as content title.

### What version of Plone/ Addons I am using:

Plone Classic 6.0.13

**Comment by rohnsha0** [comment]
confirmed... also the title doesnt change in the root_contents when navigating back using breadcrumbs

**Comment by petschki** [comment]
This is because the pattern `pat-structure-updater` isn't loaded anymore. This is responsible for updateing the title and description on the `context-info-loaded` event which is triggered by `pat-structure` ... see https://github.com/plone/mockup/blob/master/src/pat/structure-updater/structure-updater.js

**Comment by rohnsha0** [comment]
i am unsure, how to fix the same!

**Comment by rohnsha0** [comment]
then this should better be handled in `repo: mockups`?

**Comment by petschki** [comment]
Its ok to create the issue here ... I'll reference to it when fixing it in `mockup` ...

**Comment by stevepiercy** [comment]
@rohnsha0 this repo is the default when you don't know in which repo the issue should be created. Someone will help identify in which of the dozens of repos that make up Plone the actual change and pull request should be made.

And I just realized that we don't document this practice in [Contributing to Plone](https://6.docs.plone.org/contributing/index.html). I created an issue for that in https://github.com/plone/documentation/issues/1699.
